### ISSUE TITLE
Instagram Icon

### ISSUE THREAD

**Issue by nseb** [akveo/eva-icons#80]
Hello,
i don't find the instagram icon , perhaps facebook ,twitter ,google are presents
Best Regards

**Comment by nseb** [comment]
Up

**Comment by RWOverdijk** [comment]
There's a linkedIn icon, but no instagram. Confirmed: eva design is for old people.

All joking aside, can we help? There are plenty of existing icons out there so it shouldn't be too hard to add an icon, right?

**Comment by Eshaan7** [comment]
We also need a slack icon.

**Comment by adeolaex** [comment]
True. So far this has been my best Icon pack. Would love to see some core icons  :)

**Comment by rafaelcavalcante** [comment]
> There's a linkedIn icon, but no instagram. Confirmed: eva design is for old people.
> 
> All joking aside, can we help? There are plenty of existing icons out there so it shouldn't be too hard to add an icon, right?

If we manage to make the project run locally we can fork and manage it.

**Comment by joeljerushan** [comment]
still not available Instagram, youtube etc 👎

**Comment by RWOverdijk** [comment]
I just register custom icons now. I stopped using eva-icons and it gives more freedom. Designing and adding icons is a lot of work, so your best bet is to use a different icon library or adding [your own adapter](https://akveo.github.io/react-native-ui-kitten/docs/guides/icon-packages#3rd-party-icon-packages) with your own/third party icons.

**Comment by rdewolff** [comment]
Would love to see an instagram icon here too 👀

**Comment by rafaelcavalcante** [comment]
This project is abandoned 👍
